doc/dev/index.rst

```diff
@@ -628,25 +709,51 @@ is as follows::
       directory: tasks
          file: ceph-disk.yaml
 
-This is interpreted as two tests:
+This is interpreted as two tests: (1) the concatenation of centos_7.0.yaml
+and ceph-disk.yaml, and (2) the concatenation of ubuntu_14.04.yaml and
+ceph-disk.yaml. In accordance with teuthology usage, the test descriptions
+are:
 
-* the concatenation of centos_7.0.yaml and ceph-disk.yaml
-* the concatenation of ubuntu_14.04.yaml and ceph-disk.yaml
+1. ceph-disk/basic/{distros/centos_7.0.yaml tasks/ceph-disk.yaml}
+2. ceph-disk/basic/{distros/ubuntu_14.04.yaml tasks/ceph-disk.yaml}
 
-Meaning the task found in ``ceph-disk.yaml`` is intended to run on
-both CentOS 7.0 and Ubuntu 14.04.
+(In human terms, this means that the task found in ``ceph-disk.yaml`` is
+intended to run on both CentOS 7.0 and Ubuntu 14.04.)
+
+If you look up these files in the `ceph-qa-suite`_ repo, you will notice
+that the ``centos_7.0.yaml`` and ``ubuntu_14.04.yaml`` files in the
+``suites/ceph-disk/basic/distros/`` directory are implemented as symlinks
+to ``distros/centos_7.0.yaml`` and ``distros/ubuntu_14.04.yaml``,
+respectively. The practice of using symlinks instead of files enables a
+single file to be used in multiple suites.
 
 The special file percent (``%``) is interpreted as a requirement to
 generate tests combining all files found in the current directory and
 in its direct subdirectories. Without the file percent, the
-``ceph-disk`` tree would create three independant tests:
+``ceph-disk`` tree would be interpreted as three standalone tests:
 
 * ceph-disk/basic/distros/centos_7.0.yaml
 * ceph-disk/basic/distros/ubuntu_14.04.yaml
-* ceph-disk/basic/distros/ceph-disk.yaml
+* ceph-disk/basic/tasks/ceph-disk.yaml
+
+All the tests generated from the ``suites/ceph-disk/`` directory tree
+(also known as the "ceph-disk suite") can be run with::
+
+  teuthology-suite --suite ceph-disk
+
+Similarly, an individual test from the ceph-disk suite can be run by
+specifying its description::
 
-To share parts of the test description between suites, the special
-file plus (``+``) can be used to concatenate them. For instance::
+  teuthology-suite --suite ceph-disk/basic/{distros/ubuntu_14.04.yaml tasks/ceph-disk.yaml}
```

Comment:
I don't think that would work. --suite needs to be either a directory or a file. However the following would work

    teuthology-suite --suite ceph-disk --filter 'ceph-disk/basic/{distros/ubuntu_14.04.yaml tasks/ceph-disk.yaml}'